packages/components/document-card/stories/document-card.happo.js

```diff
@@ -8,7 +8,8 @@ storiesOf('ts-document-card', module).add('test', () => {
 	const properties = {
 		dir: { rtl: 'rtl' },
 		selected: { true: true },
-		mobileDescription: { mobileDescription: 'Nice phone you have' }
+		mobileDescription: { mobileDescription: 'Nice phone you have' },
+		cardType: { document: 'document' }
 	};
 
 	const options = {
```

Comment:
Can you please change the columns option to match the available property? So each property variation show up in single column to make it easier to see the changes 👍

Reply:
nice. ok